content/knowledge/consensus_algorithms.md

```diff
@@ -0,0 +1,314 @@
+# Consensus Algorithms
+
+This document is about explaining, at a high level, how the consensus algorithms work. After reading it, the engineer should understand the mechanism behind the consensus running Flare.
+
+## FBA and Separating Sybil Attack Resistance from Consensus by Making it a Local Selection
+
+### Consensus
+
+A consensus algorithm is an agreement system that follows a predefined set of steps in order to allow a group of participants (nodes) to reach to the same decision about a statement.
+In a blockchain network nodes repeatedly decide what the complete history of a shared ledger is, from among multiple possible versions that occasionally conflict between each other.
+This network-wide agreement allows the recipient of a crypto coin to have faith that the coin is both (a) valid (not counterfeit) and (b) not already spent elsewhere.
+
+### Federated Byzantine Agreement
+
+#### Introduction
+
+Any agreement system in a distributed computing network needs to be fault-tolerant.
+This means it must produce consistent results despite errors like slow communication links, unresponsive nodes and misordered messages.
+A Byzantine agreement system is additionally tolerant of [`Byzantine`](https://medium.com/coinmonks/a-note-from-anthony-if-you-havent-already-please-read-the-article-gaining-clarity-on-key-787989107969) faults: nodes that give false information, whether due to error or in a deliberate attempt to subvert the system or gain some advantage.
+Such a system prevents the double spending problem by using a form of majority called `quorum`.
```

Comment:
```suggestion
Such a system prevents the double spending problem by using a form of majority called _quorum_.
```

Also _quorum_ is a common English word which is used in many other contexts, so I don't know about the phrasing.

WDYT of?

```suggestion
Such a system prevents the double spending problem by relying on a quorum reaching consensus.
```

Reply:
Quorum is specifically used in the consensus and has its own context in it, that is why I used it

Reply:
Fixed here https://github.com/optakt/optakt.github.io/pull/21/commits/f524c8d94d32f6edd6da60a9e7f74a848c44d4ce#diff-4bf8323c6426a93da1ce22be1b8528779ef6b61fdacef3e0d065f1e1dd47bea5R20
Thanks!